sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -16,6 +16,8 @@ import io.sentry.SpanStatus
 import io.sentry.TypeCheckHint.ANDROID_FRAGMENT
 import java.util.WeakHashMap
 
+private const val TRACE_ORIGIN = "auto.ui.fragment.lifecycle"
```

Comment:
```suggestion
private const val TRACE_ORIGIN = "auto.ui.fragment"
```
just to align with `.activity`